tutoraspects/asset_command_helpers.py

```diff
@@ -518,10 +510,9 @@ def _get_used_chart_dataset_uuids():
             filters = asset["metadata"].get("native_filter_configuration", [])
 
             for filter_config in filters:
-                for filter_dataset in filter_config.get("target", {}).get(
-                    "datasetUuid", []
-                ):
-                    used_dataset_uuids.add(filter_dataset)
+                for item in filter_config.get("targets", {}):
+                    if item.get("datasetUuid"):
+                        used_dataset_uuids.add(item.get("datasetUuid"))
```

Comment:
On superset 4.1.0 this will change to `dashboardId`, I didn't find how to match it against the `datasetUuid`. Anyway, problem for future me

Reply:
What is changing to dashboardId? The dataset is definitely different from dashboard.

Reply:
Sorry, `datasetId` I mean